server/initializers/sentry.js

```diff
@@ -0,0 +1,47 @@
+const { Initializer, api } = require('actionhero')
+const Sentry = require('@sentry/node')
+
+module.exports = class SentryInit extends Initializer {
+  constructor () {
+    super()
+    this.name = 'sentry'
+    this.loadPriority = 500
+  }
+
+  async initialize () {
+    if (api.config.sentry.dsn) {
+      Sentry.init({ dsn: api.config.sentry.dsn })
+
+      api.log('Sentry initialized with', 'info', api.config.sentry)
+
+      // create a custom error reporter that sends error to sentry
+      var sentryErrorReporter = function (err, type, name, objects, severity) {
+        var options = {
+          level: severity,
+          extra: {
+            error_type: type,
+            name: name
+          }
+        }
+
+        if (type === 'action') {
+          var user = { ip: objects.connection.remoteIP }
+          if (objects.connection.user) {
+            user.id = objects.connection.user.id
+            user.email = objects.connection.user.email
+          }
+
+          options.user = user
+          options.params = objects.connection.params
+        }
+
+        Sentry.captureException(err)
```

Comment:
Don't see any use of `options` - maybe just remove it, or it may need to be added as `extra` or `tags`